packages/@react-stately/table/src/useTableState.ts

```diff
@@ -35,9 +55,16 @@ export interface CollectionBuilderContext<T> {
   columns: Node<T>[]
 }
 
-export interface TableStateProps<T> extends CollectionBase<T>, MultipleSelectionStateProps, Sortable {
+export interface TableStateProps<T>
+  extends CollectionBase<T>,
+    MultipleSelectionStateProps,
+    Sortable {
   /** Whether the row selection checkboxes should be displayed. */
-  showSelectionCheckboxes?: boolean
+  showSelectionCheckboxes?: boolean,
+  /** Function for determining the default width of columns. */
+  getDefaultWidth: (props) => string | number,
```

Comment:
We actually realized this would be a breaking API change if we didn't for anyone using useTableState and so we did change that to be optional down the chain.